test/integration/targets/get_certificate/tests/validate.yml

```diff
@@ -0,0 +1,90 @@
+---
+- name: Test get_certificate
+  connection: local
+  hosts: localhost
+  gather_facts: false
+  tasks:
+
+    - name: Get red hats certificate
+      get_certificate:
+        host: www.redhat.com
```

Comment:
I don't know what the policy is around testing against online targets...

Reply:
Nor I. There are no servers (like a Tower server) that are available at testing time are there?